performance_regression_row_level_repair_test.py

```diff
@@ -0,0 +1,483 @@
+            self.log.debug('{} Created'.format(keyspace_name))
+            table_name = "{}.standard1".format(keyspace_name)
+            self.create_table(name=table_name, key_type='blob', read_repair=0.0, compact_storage=True,
+                              columns={'"C0"': 'blob'},
+                              in_memory=in_memory, scylla_encryption_options=scylla_encryption_options)
+
+    def _update_cl_in_stress_cmd(self, str_stress_cmd, cl):
+        for param in str_stress_cmd.split():
+            if param.startswith('cl='):
+                return str_stress_cmd.replace(param, "cl={}".format(cl))
+        self.log.debug("Could not find a 'cl' parameter in stress command: {}".format(str_stress_cmd))
+        return str_stress_cmd
+
+    def preload_data(self, cl=None):
+        # if test require a pre-population of data
+
+        prepare_write_cmd = self.params.get('prepare_write_cmd')
+        if prepare_write_cmd:
+            self.create_test_stats(sub_type='write-prepare')
+            stress_queue = list()
+            params = {'prefix': 'preload-'}
+            # Check if the prepare_cmd is a list of commands
+            if not isinstance(prepare_write_cmd, basestring) and len(prepare_write_cmd) > 1:
+                # Check if it should be round_robin across loaders
+                if self.params.get('round_robin', default='').lower() == 'true':
+                    self.log.debug('Populating data using round_robin')
+                    params.update({'stress_num': 1, 'round_robin': True})
+
+                for stress_cmd in prepare_write_cmd:
+                    if cl:
+                        stress_cmd = self._update_cl_in_stress_cmd(str_stress_cmd=stress_cmd, cl=cl)
+                    params.update({'stress_cmd': stress_cmd})
+
+                    # Run all stress commands
+                    params.update(dict(stats_aggregate_cmds=False))
+                    self.log.debug('RUNNING stress cmd: {}'.format(stress_cmd))
+                    stress_queue.append(self.run_stress_thread(**params))
+
+            # One stress cmd command
+            else:
+                stress_cmd = prepare_write_cmd if not cl else self._update_cl_in_stress_cmd(
+                    str_stress_cmd=prepare_write_cmd, cl=cl)
+                stress_queue.append(self.run_stress_thread(stress_cmd=stress_cmd, stress_num=1,
+                                                           prefix='preload-', stats_aggregate_cmds=False))
+
+            for stress in stress_queue:
+                self.get_stress_results(queue=stress, store_results=False)
+
+            self.update_test_details()
+        else:
+            self.log.warning("No prepare command defined in YAML!")
+
+    # Util functions ===============================================================================================
+
+
+    # Global Util functions ===============================================================================================
```

Comment:
what's the difference between `utils functions` to `global utils functions` ?

Reply:
Would love to see the those global one goes into tester.py, or to their own file

Reply:
moved some utils to tester.py . left only the 'private util functions'